pkg/xen-tools/Dockerfile

```diff
@@ -89,10 +91,13 @@ RUN mkdir -p /out
 
 # FEATURES="--enable-stubdom --enable-vtpm-stubdom --enable-vtpmmgr-stubdom"
 COPY gmp.patch /xen/stubdom
+
 WORKDIR /xen
+
 # disable golang as it does not play well together with musl (stderr is defined as FILE* const and fails to compile)
 RUN ./configure --prefix=/usr --disable-xen --disable-golang --disable-qemu-traditional --disable-docs --enable-9pfs \
-                --with-system-ovmf=/usr/lib/xen/boot/ovmf.bin --disable-stubdom
+ --with-system-ovmf=/usr/lib/xen/boot/ovmf.bin --disable-stubdom \
```

Comment:
please add the extra spaces back, increases the readability. same for `EXTRA_QEMUU_CONFIGURE_ARGS`.